Testing/ORKTest/ORKTest/TaskFactory.h

```diff
@@ -0,0 +1,93 @@
+//
+//  TaskFactory.h
+//  ORKTest
+//
+//  Created by Ricardo Sanchez-Saez on 1/9/17.
+//  Copyright © 2017 ResearchKit. All rights reserved.
+//
+
+#import <Foundation/Foundation.h>
+
+#define DefineStringKey(x) static NSString *const x = @#x
+
+DefineStringKey(ConsentTaskIdentifier);
```

Comment:
Sorry for the confusion. I meant that there aren't that many PRs right now that are affected by this so it is a good time to do it and yes, I think it's better to do the alphabetizing when you are moving the code b/c you've already created the new file and any new PRs will need to be changed to use the factory anyway.